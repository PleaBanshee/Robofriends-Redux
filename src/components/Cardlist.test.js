import { shallow } from 'enzyme';
import CardList from './CardList';


it('CardList component must render', () => {
    expect.assertions(1);
    const mockRobots = [
        {
            id: 1,
            name: 'John Snow',
            username: 'GOTF@n',
            email: 'john@gmail.com'
        }

    ];
    expect(shallow(<CardList robots={mockRobots}/>)).toMatchSnapshot();
});