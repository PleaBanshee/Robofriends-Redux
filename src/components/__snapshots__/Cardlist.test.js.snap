// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardList component must render 1`] = `
<div>
  <Card
    email="john@gmail.com"
    id={1}
    key="0"
    name="John Snow"
  />
</div>
`;
